org.gnome.Evince.json

```diff
@@ -10,7 +10,10 @@
         "--socket=x11",
         "--socket=wayland",
         "--socket=pulseaudio",
-        "--filesystem=host",
+        "--filesystem=/media:ro",
+        "--filesystem=/run/media:ro",
+        "--filesystem=xdg-documents",
```

Comment:
Why does it need access to xdg-documents and xdg-download if it can access those files through the document portal in the first place?